extensions/wikia/FacebookClient/FacebookClient.class.php

```diff
@@ -281,6 +281,19 @@ public function getWikiaUser( $fbId = null ) {
 			return null;
 		}
 
+		// Create a new mapping that includes the app ID.  Leave the default App ID behind
+		// as there is no way to tell what other apps this user had connected, and removing this
+		// default mapping will force the user to reconnect on those apps.
+		if ( $map->isDefaultAppId() ) {
+			$this->migrateMapping( $map );
+		}
+
```

Comment:
This process starts updating existing user's Business Token.  This token is not currently used, but tells us which users are the same users across FB Apps now that FB User IDs are unique only within an App.